x-pack/solutions/search/packages/kbn-search-code-examples/src/api-tutorials/time_series_data_stream_tutorial.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export const timeSeriesDataStreamTutorialCommands: string = `# Welcome to the tutorial on setting up time series data! 🎉
+# 🚀 This tutorial will guide you through setting up timeseries data stream, ingest sample data and run a query from the Kibana dev Console.
+# After selecting a command, execute it by clicking the ▶️ button or pressing Ctrl+Enter or Cmd+Enter.
+# A time series data stream (TSDS) stores timestamped data and one or more metrics data in real time. TSDS helps to store data at regular interval which can then used to real time monitoring.
+# -----------------------------------------------
+# Step 1: Create an index template
+# Requirement in matching index template:
+#  * a "data_stream" object
+#  * index.mode field set as "time_series"
+#  * Atleast one timestamp field with type date
```

Comment:
This should be "At least"

Reply:
updated 2393fd105f5f0cfe281014457616024bcbe71e2c